### PR TITLE
V1 Toggle Component

### PR DESCRIPTION
**Pull request by renatorib**
**Props**
_on, toggle, set_

`on`
`toggle()`
`set(true)` | `set(on => !on)`

### CODE REVIEW

**Comment by renatorib** [comment]
Hey @TrySound, was I said, I do not have knowledge with flow. Can you help me checking if I'm doing nothing wrong? Thanks!

File: src/utils/set.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const set = (updater, arg) =>
+  typeof updater === 'function' ? updater(arg) : updater
+
+export default set
```
Comment (TrySound) [inline src/utils/set.js]: Are you gonna reuse it? Where?

**Comment by TrySound** [comment]
Looks good

File: src/utils/set.js (hunk 1 above)
Comment (renatorib) [reply src/utils/set.js]: everywhere xD

all components will have set function that will work as this

see https://github.com/renatorib/react-powerplug/issues/6

**Pull request merged**